site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -163,7 +163,45 @@ When an API key is used that is locked to a Tenant or the tenant `X-FusionAuth-T
 [field]#users``[x]``.timezone# [type]#[String]# [optional]#Optional#::
 The User's preferred timezone. The string must be in an https://www.iana.org/time-zones[IANA] time zone format.
 
-[field]#users``[x]``.twoFactorDelivery# [type]#[String]# [optional]#Optional# [default]#defaults to `None`#::
+[field]#users``[x]``.twoFactor.methods``[x]``.authenticator.algorithm# [type]#[TOTPAlgorithm]# [optional]#Optional#::
+The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`.
```

Comment:
"The algorithm used by the TOTP authenticator. With the current implementation, this will always be `HmacSHA1`." =>  "The algorithm used by the TOTP authenticator. With the current implementation, this must be `HmacSHA1` if provided. Any other value will be ignored."

Since this is a request, correct?

Reply:
Good call, that is correct.